src/redisearch_api.c

```diff
@@ -247,6 +251,9 @@ int RediSearch_IndexAddDocument(IndexSpec* sp, Document* d, int options, char**
 }
 
 QueryNode* RediSearch_CreateTokenNode(IndexSpec* sp, const char* fieldName, const char* token) {
+  if (StopWordList_Contains(sp->stopwords, token, strlen(token))) {
+    return NULL;
+  }
```

Comment:
This is what fixes RedisGraph/RedisGraph#1638 issue

Reply:
Can you please show me to use this in the RG commands? I cannot find documentation for this and I have tried guessing and checking several items with no success.

Reply:
@sidenAlex
You might find our unit tests helpful. ([link](https://github.com/RediSearch/RediSearch/blob/b19e16f58fb8cb867444755710d55f2cd6107898/tests/cpptests/test_cpp_llapi.cpp#L824))